src/Uvweb/UvBundle/Controller/HomeController.php

```diff
@@ -90,10 +90,12 @@ public function getUvNamesLikeAction()
 
         //Keeping only the uv names
         foreach ($uvs as $uv) {
-            $uvNames[] = $uv['name'];
+            $uvNames[] = array('name' => $uv['name'], 'uni' => $uv['uni'], 'uniId' => $uv['uniId']);
         }
 
-        return new Response(json_encode($uvNames));
+        $resp = new Response(json_encode($uvNames));
+        $resp->headers->set('Content-Type', 'application/json; charset=utf-8');
```

Comment:
Couldn't it be the third argument of the Response contructor?
It would be easier to read/maintain.